.travis.yml

```diff
@@ -9,29 +9,6 @@ matrix:
       script:
         make checkformatting
 
-    # Python (backend) tests
-    - env: ACTION=tox
-      language: python
-      python: '3.6'
-      services:
-        - postgresql
-      addons:
-        postgresql: "11"
-      before_install:
-        - sudo apt-get update
-        - sudo apt-get --yes remove postgresql\*
-        - sudo apt-get install -y postgresql-11 postgresql-client-11
-        - sudo cp /etc/postgresql/{9.6,11}/main/pg_hba.conf
-        - sudo service postgresql restart 11
-        - ./scripts/elasticsearch.sh
```

Comment:
The `scripts/elasticsearch.sh` file can be deleted too

Reply:
Righto. I've scooped that out now.